Comparing source compatibility of C:\au\builds\uimaj-3.1.0\rc1\artifacts\target\checkout\uimaj-core\target\uima-core.jar against C:\Users\MarshallSchor\.m2\repository\org\apache\uima\uimaj-core\3.0.2\uimaj-core-3.0.2.jar
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.CAS  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++! NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FSArray emptyFSArray(org.apache.uima.cas.Type)
***  MODIFIED CLASS: PUBLIC org.apache.uima.InternationalizedException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) void setResourceBundle(java.util.Locale)
***  MODIFIED CLASS: PUBLIC org.apache.uima.InternationalizedRuntimeException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW METHOD: PUBLIC(+) void setResourceBundle(java.util.Locale)
***  MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.FSArray  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW CONSTRUCTOR: PUBLIC(+) FSArray(java.lang.Class, org.apache.uima.jcas.JCas, int)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.jcas.JCas  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++! NEW METHOD: PUBLIC(+) org.apache.uima.jcas.cas.FSArray emptyFSArray(java.lang.Class)
